tests/test_grpc.py

```diff
@@ -513,4 +492,4 @@ def test_start_participant(mock_coordinator_service):
         assert coord.current_round == 1
 
         # expect weight aggregated by summation - see mock_coordinator_service
-        np.testing.assert_equal(coord.weights, [np.arange(start=10, stop=29, step=2)])
+        np.testing.assert_equal(coord.weights, init_weight)
```

Comment:
now that weights are flat, there is no longer an interesting aggregate result to test for, which is a bit of a shame. but i would not necessarily consider that part of this PR!

Reply:
I will create a separate ticket for this